# Large amplitude wave path - more aggressive maneuvers

waypoints:
  # Large wave (bigger amplitude, wider turns)
  - [0.0, 0.0]
  - [7.0, 5.0]
  - [14.0, 0.0]
  - [21.0, -5.0]
  - [28.0, 0.0]
  - [35.0, 5.0]
  - [42.0, 0.0]
  - [49.0, -5.0]
  - [56.0, 0.0]
  - [63.0, 5.0]
  - [70.0, 0.0]

path_settings:
  interpolation: 'cubic'
  resolution: 50
  closed_path: false
